index.js

```diff
@@ -6,59 +6,137 @@ module.exports = {
 	parserOptions: {
 		ecmaVersion: 12,
 	},
-	plugins: ['import'],
+	plugins: ['simple-import-sort', 'unused-imports'],
 	rules: {
 		'arrow-body-style': ['off'],
 		'consistent-return': ['off'],
-		'import/extensions': ['error', 'ignorePackages', {
-			js: 'never',
-			jsx: 'never',
-			mjs: 'never',
-			ts: 'never',
-			tsx: 'never',
-		}],
+		'import/extensions': [
+			'error',
+			'ignorePackages',
+			{
+				js: 'never',
+				jsx: 'never',
+				mjs: 'never',
+				ts: 'never',
+				tsx: 'never',
+			},
+		],
 		'import/no-extraneous-dependencies': ['off'],
 		'import/no-named-as-default': ['off'],
-		'import/order': ['error', {
-			alphabetize: {
-				order: 'asc',
-			},
-			groups: [
-				'builtin',
-				'external',
-				'type',
-				'internal',
-				'parent',
-				'sibling',
-				'index',
-				'object',
-			],
-			'newlines-between': 'always',
-		}],
+		'unused-imports/no-unused-imports': 'error',
+		'unused-imports/no-unused-vars': [
+			'warn',
+			{
+				vars: 'all',
+				varsIgnorePattern: '^_',
+				args: 'after-used',
+				argsIgnorePattern: '^_',
+			},
+		],
+		'simple-import-sort/imports': [
+			'warn',
+			{
+				groups: [
+					['^((?!((.*).(scss|css))|assets|src|./|../|@[^lingui|^storybook|^sentry]|^tailwind.config.js))'],
+					['^(@[^lingui|^storybook|^sentry](.*))'],
+					['^(@apps/|@libs/)'],
+					['^(components/)'],
+					['^(config/)'],
+					['^(lib/)'],
+					['^(pages/)'],
+					['^(state/)'],
+					['^(styles/)'],
+					['^(translations/)'],
+					['^(types/)'],
+					['^(utils/)'],
+					['^(tailwind.config.js|(.*).(scss|css)$)'],
+					['^((./|../))'],
+				],
+			},
+		],
```

Comment:
This is **very** project specific. I think this is an overwrite which should be added in specific projects if needed/wanted. I think leaving the current `import/order` rule in place as a default would be best.